classes/PodsInit.php

```diff
@@ -1091,6 +1091,16 @@ public function setup_content_types( $force = false ) {
 					'sort'                  => (boolean) pods_v( 'sort', $taxonomy, false ),
 				);
 
+				// @since WP 5.5: Default terms.
+				$default_term_name = pods_v( 'default_term_name', $taxonomy, null, true );
+				if ( $default_term_name ) {
+					$pods_taxonomies[ $taxonomy_name ][ 'default_term' ] = array(
```

Comment:
:no_entry_sign: Array keys must NOT be surrounded by spaces if they only contain a string or an integer.

<sub>hash: 892cd4bfa54152fa504a6ebe89f7973c</sub>